lib/server.js

```diff
@@ -110,7 +110,9 @@ class S3Server {
                 log.info('initial health check succeeded', {
                     healthStatus: results,
                 });
-                this.startup();
+                // creates dual-stack endpoints for IPv4 and IPv6
+                this.startup('0.0.0.0', 8000);
```

Comment:
We would still want this to be configurable, so that Federation can change the ports if needed. Having something like

```js
this.startup('0.0.0.0', _config.port);
this.startup('::', _config.ipv6Port);
```
would help.